app/ante/eth.go

```diff
@@ -519,7 +519,7 @@ func (mfd EthMempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulat
 		params := mfd.evmKeeper.GetParams(ctx)
 		ethCfg := params.ChainConfig.EthereumConfig(mfd.evmKeeper.ChainID())
 		baseFee := mfd.evmKeeper.GetBaseFee(ctx, ethCfg)
-		if baseFee == nil {
+		if baseFee == nil || baseFee.BitLen() == 0 {
```

Comment:
Updated to match ```// It only do the check if london hardfork not enabled or feemarket not enabled, because in that case feemarket will take over the task.```

London hardfork not enabled (baseFee == nil)
Feemarket not enabled (baseFee == 0)

Reply:
An alternative solution might be to return `nil` in `evmKeeper.GetBaseFee` when feemarket is not enabled?